src/schemathesis/runner/impl/core.py

```diff
@@ -235,11 +235,13 @@ def run_targets(targets: Iterable[Target], elapsed: float) -> None:
             hypothesis.target(elapsed, label="response_time")
 
 
-def add_cases(case: Case, test: Callable, *args: Any) -> None:
+def add_cases(case: Case, case_response: Union[requests.Response, WSGIResponse], test: Callable, *args: Any) -> None:
```

Comment:
`schemathesis.utils.GenericResponse` can be used here for `case_response`. Also, what do you think about using `response` name instead of `case_response`?

Reply:
I like both ideas!